src/python/grpcio/grpc/_server.py

```diff
@@ -560,7 +570,22 @@ def _handle_unrecognized_method(rpc_event):
     return rpc_state
 
 
-def _handle_with_method_handler(rpc_event, method_handler, thread_pool):
+def _reject_call(rpc_event):
+    operations = (cygrpc.operation_send_initial_metadata(_common.EMPTY_METADATA,
+                                                         _EMPTY_FLAGS),
+                  cygrpc.operation_receive_close_on_server(_EMPTY_FLAGS),
+                  cygrpc.operation_send_status_from_server(
+                      _common.EMPTY_METADATA,
+                      cygrpc.StatusCode.resource_exhausted,
+                      b'Max requests exceeded!', _EMPTY_FLAGS),)
```

Comment:
I've left out the the integer limit.  This feels like we are exposing too much server information to the client, even if it is not a security risk.

The service owner should be aware of the limit because it needs to be set explicitly.